src/sap.m/test/sap/m/demokit/helloworld/index.html

```diff
@@ -1,54 +1,60 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-	<meta charset="UTF-8">
-	<title>Hello World App</title>
-	<script
-		id="sap-ui-bootstrap"
-		src="../../../../../../../../resources/sap-ui-core.js"
-		data-sap-ui-theme="sap_bluecrystal"
-		data-sap-ui-libs="sap.m"
-		data-sap-ui-preload="async">
-	</script>
-	<script type="text/javascript">
-		sap.ui.getCore().attachInit(function () {
-			// create a mobile app and display page1 initially
-			var app = new sap.m.App("myApp", {
-				initialPage: "page1"
-			});
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
The indention seems to be wrong. There are two tabs here, but <head> does not have any indention.